src/api_manager/fetch_metadata.cc

```diff
@@ -96,7 +96,7 @@ void GlobalFetchGceMetadata(std::shared_ptr<context::GlobalContext> context,
         if (status.ok()) {
           status = context->gce_metadata()->ParseFromJson(&body);
         } else {
-          status = Status(Code::INTERNAL, kFailedMetadataFetch);
```

Comment:
maybe log a error message?  and please add comment 
// Fetching GceMetadata is optional,  if it fails, not to fail the user requests.

Reply:
Done